docs/blog/2024-07-17-release-1.0.0.md

```diff
@@ -24,6 +24,8 @@ your nullness analyzer is to support JSpecify, and whether you or your users
 also use Kotlin.)
 
 We've updated our [User Guide](/docs/user-guide) and our
-[Javadoc](https://jspecify.dev/docs/api/). We're still working on updating the
-[specification](/docs/spec) to reflect all of our design decisions (ETA: end of
-August), but that's more useful to nullness analyzer implementers.
+[Javadoc](https://jspecify.dev/docs/api/).
+
+October update: We've now also updated our [specification](/docs/spec) to
```

Comment:
Sneaky!

What about leaving in the old sentence but ~striking it out~?

Reply:
Done.

I then iteratively shortened this new sentence. I may have gone too far, and I could tack "for 1.0.0" or something back on if you think it would help.

Reply:
all good